app/tests/algorithms_tests/test_models.py

```diff
@@ -81,3 +82,33 @@ def test_algorithm_job_update_status():
             in email.body
         )
     assert remaining_recipients == set()
+
+
+@pytest.mark.django_db
+def test_default_interfaces_created():
+    a = AlgorithmFactory()
+
+    assert {i.kind for i in a.inputs.all()} == {InterfaceKindChoices.IMAGE}
+    assert {o.kind for o in a.outputs.all()} == {InterfaceKindChoices.JSON}
+
+
+@pytest.mark.django_db
+def test_outputs_are_set():
+    j = AlgorithmJobFactory()
```

Comment:
not used I think?

Reply:
It's there because you want to test that you're picking up the correct interface (in the first implementation I was updating the same CIV for each job, oops!). I will add a comment.